docs/user/interop.rst

```diff
@@ -308,7 +308,12 @@ runtime system, one has to be extra careful when working with unmanaged memory.
 
       import scala.scalanative.unsafe._
 
-      Zone { implicit z =>
+      // For Scala 3
+      Zone {
+        val buffer = alloc[Byte](n)
+      }
+      // For Scala 2.13
```

Comment:
Perhaps something like
`// For Scala 2.13, works but is not idiomatic on Scala 3`